spec/javascript/components/supplemental_application/__snapshots__/SupplementalApplicationPage.test.js.snap

```diff
@@ -594,17 +594,24 @@ Array [
                 <div
                   className="form-group"
                 >
-                  <label
-                    htmlFor=""
+                  <div
```

Comment:
this is a larger number of changes than I expected, but skimming through none of the changes look out of place, and the actual visual interaction looks correct